phx/events/tests/models/test_events_model.py

```diff
@@ -0,0 +1,7 @@
+from django.test import TestCase
+
+
+class TestEventsModel(TestCase):
+
+    def test_placeholder(self):
+        self.assertEqual(1, 1)
```

Comment:
My favourite kind of test 🙂 

(Fine to leave in but also no need to add one if we can't think of anything useful to test...)

Reply:
I think I add some later :)